Frontend/ui-driver/src/MerchantConfigs/JatriSathiConfig.js

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
any specific reason why we dont have all these files in `.purs`

Reply:
Yes, Currently this config is a json file, If we define json in .purs the we have to define every type.